src/archivematicaCommon/tests/test_execute_functions.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: UTF-8 -*-
+
+import executeOrRunSubProcess as execsub
+
+
+def test_capture_output():
+    """Tests behaviour of capture_output when executing sub processes."""
+
+    # Test that stdout and stderr are not captured by default
+    ret, std_out, std_err = execsub.launchSubProcess(['ls', '/tmp'])
+    assert std_out is ''
+    assert std_err is ''
+
+    # Test that stdout and stderr are captured when `capture_output` is
+    # enabled.
+    ret, std_out, std_err = execsub.launchSubProcess(
+        ['ls', '/tmp'], capture_output=True)
+    assert std_out is not '' or std_err is not ''
+
+    # Test that stdout and stderr are not captured when `capture_output` is
+    # not enabled.
+    ret, std_out, std_err = execsub.launchSubProcess(
+        ['ls', '/tmp'], capture_output=True)
```

Comment:
I was able to run the tests as follows:
```
COMPOSE_FILE=docker-compose.dev.yml:docker-compose.qa.yml docker-compose run --rm --no-deps --entrypoint py.test --workdir /src/ archivematica-dashboard archivematicaCommon/tests/test_execute_functions.py
```

(running a full test suite causes other failures and I just wanted to focus on this one).

I think this one should be `capture_output=False`. (to match the comment and also make the test pass). Otherwise looks good :)